src/Client.php

```diff
@@ -494,6 +545,26 @@ protected function getFileIdFromBucketAndFileName($bucketName, $fileName)
         }
     }
 
+    /**
+     * @param $bucketId
+     * @param $fileName
+     *
+     * @throws B2Exception
+     * @throws GuzzleException
+     *
+     * @return string|bool The file's id, or false if not found
+     */
+    protected function getFileIdFromBucketIdAndFileName($bucketId, $fileName)
```

Comment:
You can remove this function altogether and use the below instead. I actually made the same mistake myself not realising that `listFiles` will search for one file if you specify its name.
```php
$files = $this->listFiles([
    'BucketId' => $bucketId,
    'FileName' => $fileName,
]);

$fileId = !empty($files) ? $files[0]->getId() : false;
```